### PR TITLE
[ES] Add "Ranking"

### PR DESCRIPTION
**Pull request by Ekkusu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Review by agatemosu — CHANGES_REQUESTED** [review]

File: wiki/Ranking/es.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+La clasificación de puntos de rendimiento describe el nivel de habilidad de un jugador en comparación con otros jugadores. Esta tabla de clasificación se determina por el número total de puntos de rendimiento que tiene un jugador individual, y se puede ver en el perfil de un jugador y en las [tablas de rendimiento](https://osu.ppy.sh/rankings/osu/performance).
+
+## Clasificación de puntuación
+
+La clasificación de puntuación compara la puntuación total de los jugadores y los enumera en consecuencia. Ocupar un lugar destacado en esta clasificación requiere un gran compromiso de tiempo, pero se necesita relativamente menos habilidad mecánica que en la clasificación de puntos de rendimiento.
+
+## Clasificación de Kudosu
+
+La clasificación de Kudosu compara los [Kudosu](/wiki/Modding/Kudosu) acumulados por los usuarios y los enumera en consecuencia. Esta clasificación no requiere habilidades de juego, ya que los Kudosu se obtienen como parte del [proceso de modding](/wiki/Modding).
```
Comment (UnknownOsu) [inline wiki/Ranking/es.md:32]:
```suggestion
La clasificación por Kudosu compara el [Kudosu](/wiki/Modding/Kudosu) acumulado por los usuarios y los enumera en consecuencia. Esta clasificación no requiere habilidades de juego, ya que el Kudosu se obtiene como parte del [proceso de modding](/wiki/Modding).
```

File: wiki/Ranking/es.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+## Clasificación de puntos de rendimiento
+
+*Véase también: [Puntos de rendimiento](/wiki/Performance_points)*
+
+La clasificación de puntos de rendimiento describe el nivel de habilidad de un jugador en comparación con otros jugadores. Esta tabla de clasificación se determina por el número total de puntos de rendimiento que tiene un jugador individual, y se puede ver en el perfil de un jugador y en las [tablas de rendimiento](https://osu.ppy.sh/rankings/osu/performance).
+
+## Clasificación de puntuación
+
+La clasificación de puntuación compara la puntuación total de los jugadores y los enumera en consecuencia. Ocupar un lugar destacado en esta clasificación requiere un gran compromiso de tiempo, pero se necesita relativamente menos habilidad mecánica que en la clasificación de puntos de rendimiento.
```
Comment (UnknownOsu) [inline wiki/Ranking/es.md:28]:
```suggestion
La clasificación por puntuación compara la puntuación total de los jugadores y los enumera en consecuencia. Ocupar un lugar destacado en esta clasificación requiere un gran compromiso de tiempo, pero se necesita relativamente menos habilidad mecánica que en la clasificación por puntos de rendimiento.
```

File: wiki/Ranking/es.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+La **clasificación** (alternativamente llamada «tablas de clasificación») es una lista que ordena a los miembros de la comunidad de osu! según ciertos criterios.
+
+Todas las clasificaciones, excepto la clasificación individual de beatmaps, se pueden acceder pasando el cursor sobre el menú `clasificaciones` en el encabezado del sitio web. Para las clasificaciones individuales de beatmaps, se pueden encontrar en su página de información del beatmap.
+
+## Clasificación de puntos de rendimiento
+
+*Véase también: [Puntos de rendimiento](/wiki/Performance_points)*
+
+La clasificación de puntos de rendimiento describe el nivel de habilidad de un jugador en comparación con otros jugadores. Esta tabla de clasificación se determina por el número total de puntos de rendimiento que tiene un jugador individual, y se puede ver en el perfil de un jugador y en las [tablas de rendimiento](https://osu.ppy.sh/rankings/osu/performance).
```
Comment (UnknownOsu) [inline wiki/Ranking/es.md:24]:
```suggestion
La clasificación por puntos de rendimiento describe el nivel de habilidad de un jugador en comparación con otros jugadores. Esta tabla de clasificación se determina por el número total de puntos de rendimiento que tiene un jugador individual, y se puede ver en el perfil de un jugador y en las [tablas de rendimiento](https://osu.ppy.sh/rankings/osu/performance).
```

File: wiki/Ranking/es.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+## Clasificación de Spotlights
+
+Las clasificaciones de Spotlights son tablas de clasificación de tiempo limitado que comparan la habilidad de los jugadores en un conjunto específico de beatmaps en cada temporada de [Spotlights](/wiki/Beatmap_Spotlights). Los jugadores que se clasifiquen alto en las clasifiacaciones de Spotlights recibirán premios específicos como resultado de su arduo trabajo.
+
+## Clasificación de beatmaps
+
+Las clasificaciones de beatmaps comparan las puntuaciones de los jugadores entre sí en beatmaps individuales. Aparecen una vez que una beatmap ha sido [calificado](/wiki/Beatmap/Category#calificados), [clasificado](/wiki/Beatmap/Category#clasificados), [aprobado](/wiki/Beatmap/Category#aprobados), o Amado. Las ocho mejores puntuaciones se muestran sin necesidad de desplazarse, y si el jugador tiene una de las 50 mejores puntuaciones, la pantalla se desplazará automáticamente hasta su puntuación. Las clasificaciones en primer lugar se pueden ver en la página de perfil de un usuario.
+
+## Clasificación de países
```
Comment (UnknownOsu) [inline wiki/Ranking/es.md:42]:
```suggestion
## Clasificación por países
```

File: wiki/Ranking/es.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+La clasificación de países se basan en los puntos de rendimiento de los jugadores de un país.
```
Comment (UnknownOsu) [inline wiki/Ranking/es.md:44]:
```suggestion
La clasificación por países se basan en los puntos de rendimiento de los jugadores de un país.
```

File: wiki/Ranking/es.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+# Clasificación
+
+*Para otros usos, véase [Rango (desambiguación)](/wiki/Disambiguation/Rank).*
+
+La **clasificación** (alternativamente llamada «tablas de clasificación») es una lista que ordena a los miembros de la comunidad de osu! según ciertos criterios.
+
+Todas las clasificaciones, excepto la clasificación individual de beatmaps, se pueden acceder pasando el cursor sobre el menú `clasificaciones` en el encabezado del sitio web. Para las clasificaciones individuales de beatmaps, se pueden encontrar en su página de información del beatmap.
+
+## Clasificación de puntos de rendimiento
```
[the remark this answers is not in the document]
Comment (UnknownOsu) [reply wiki/Ranking/es.md:20]: Creo que queda mejor `por` en lugar de `de` en algunos casos

**Review by UnknownOsu — CHANGES_REQUESTED** [review]

**Review by agatemosu — APPROVED** [review]

**Review by cl8n — APPROVED** [review]

**Pull request merged**